block-util/src/queue/queue_diff.rs

```diff
@@ -27,11 +27,11 @@ impl QueueDiffStuffBuilder {
     }
 
     // TODO: Use iterator of `(ShardIdent, QueueKey)`?
-    pub fn with_processed_upto<'a, I>(mut self, processed_upto: I) -> Self
+    pub fn with_processed_to<'a, I>(mut self, processed_to: I) -> Self
     where
         I: IntoIterator<Item = (ShardIdent, u64, &'a HashBytes)>,
     {
-        self.inner_mut().diff.processed_upto = processed_upto
+        self.inner_mut().diff.processed_upto = processed_to
```

Comment:
Let's rename `QueueDiff.processed_upto` into `QueueDiff.processed_to` as well. But we cannot update `proto.tl` right now because type id could be changed. So we should place a `TODO` comment to the field:
```
// TODO: should rename field in `proto.tl` on network reset
```

Let's rename `mod processed_upto_map` into `mod processed_to_map`.